sidebars.js

```diff
@@ -648,7 +648,29 @@ module.exports = {
                     id: "components/connectors/out-of-the-box-connectors/agentic-ai-aiagent",
                   },
                   items: [
-                    "components/connectors/out-of-the-box-connectors/agentic-ai-aiagent-example",
+                    {
+                      type: "category",
+                      label: "AI Agent Process",
+                      link: {
+                        type: "doc",
+                        id: "components/connectors/out-of-the-box-connectors/agentic-ai-aiagent-process",
```

Comment:
https://github.com/camunda/connectors/pull/5483 is now prepared to use these URL paths